did-doc-diff/test/index.ts

```diff
@@ -0,0 +1,303 @@
+import {
+  diffs_to_did_doc,
+  sign,
+  validate_sig,
+  tick_from_diffs,
+} from '../src/index.js'
+import * as crypto from '@adxp/crypto'
+import * as uint8arrays from 'uint8arrays'
+import test from 'ava'
+import { TID } from '@adxp/common'
+
+const key_consortium = await crypto.EcdsaKeypair.import(
+  {
+    // did:key:zDnaesnjCLbxSB2EiJgTZUYCBqHTkuDJDdLbSmaAn2bpqXtuY
+    publicKey:
+      'QUwjqkzD7TZQ1iJY5TAdAD5mJDDxZrBYyi9Ld4Kebj7ZuR87TiFhKX5QeoYvffMquYTvToRV1P3EJXVqUYDMGnCc',
+    privateKey:
+      '2EPbyvKQKUaUPMF7Mm94FjEzvs5tsLWfesyc97W1dqYeeZFEG2axhDPdH5LDfLF28yNjJu26wYXiSypZtaMWCajYu4hHgUvHhygV8kHzQkvszbL6BrvGP8GsYpeWBBWBxx4q816yVUBjzFH21mkekFnHF5fWAXDUN72rii66kR81rkaxCTacSMJu2XUuz',
+  },
+  {
+    exportable: true,
+    encoding: 'base58btc',
+  },
+)
+
+const key_tick = await crypto.EcdsaKeypair.import(
```

Comment:
you're gonna hate me. i changed this import to JWK 😅

Reply:
updates to use JWT 😛